client/my-sites/upgrades/checkout/credit-card-payment-box.jsx

```diff
@@ -39,6 +40,7 @@ var CreditCardPaymentBox = React.createClass( {
 		const hasBusinessPlanInCart = some( cart.products, { product_slug: PLAN_BUSINESS } );
 		const showPaymentChatButton =
 			config.isEnabled( 'upgrades/presale-chat' ) &&
+			abtest( 'presaleChatButton' ) === 'showChatButton' &&
```

Comment:
Should these checks also include a check for `isHappychatAvailable`? Right now the button is appearing before any check is made.